ui/imports/shared/controls/chat/private/ChannelData.qml

```diff
@@ -0,0 +1,9 @@
+import QtQuick 2.15
+
+QtObject {
+    property string name
+    property string description
+    property string emoji 
+    property string color
+    readonly property CommunityData communityData: CommunityData {}
+}
```

Comment:
new line at end missing here and in other files

Reply:
That's how Mac (and Windows) works :) It's mandated on Linux only